website/src/content/docs/clustertool/virtual-machines/truenas-scale.mdx

```diff
@@ -126,12 +126,14 @@ Go back to the "preparation" section and make sure the IP you are trying to move
 
   4. You will be presented with a pane of potential options. Leave all of these as default except for `Zvol name` (call it something recognizable like `Talos zvol`), `Size for this zvol` (set to between `256GiB` and `512GiB`), `Force size` (check this option), `Sparse` (check this option) and `Block size` (change to `128 KiB`)
 
-  5. We do **not** recommend to encrypt the dataset
+  5. Set `Sync` to disabled, when you don't have enterprise-grade SSD
```

Comment:
Just tell people to keep it disabled even on enterprise. They can handle it but its better to just keep it disabled

Reply:
Very good. Had that first and included in step 4. But though better make a separate step so people don't miss it easily.